Model/U2F/TwoFactorInterface.php

```diff
@@ -3,35 +3,27 @@
 namespace R\U2FTwoFactorBundle\Model\U2F;
 
 /**
- * Interface: TwoFactorInterface
- *
+ * Interface: TwoFactorInterface.
  */
 interface TwoFactorInterface
 {
-
     /**
-     * isU2FAuthEnabled
-     * @return boolean
+     * @return bool
      **/
     public function isU2FAuthEnabled();
 
     /**
-     * getU2FKeys
      * @return array
      **/
     public function getU2FKeys();
 
     /**
-     * addU2FKey
      * @param U2FKey $key
-     * @return void
```

Comment:
Why?

Reply:
I don't see why it MUST return void. Why can't it be a fluent method?

Reply:
Being explicit about types is generally a good idea IMHO, but I'm okay either way.